v2/account_claims.go

```diff
@@ -50,6 +50,11 @@ func (n *NatsLimits) IsUnlimited() bool {
 	return *n == NatsLimits{NoLimit, NoLimit, NoLimit}
 }
 
+func (n *NatsLimits) Empty() bool {
+	// this is the default - consider that 'empty'
+	return n.IsUnlimited()
```

Comment:
I'd rather this be checked for 0. That way, looking at such a user jwt, it can be differentiated from a user that has no limits. Also no need to drag around -1 s that won't be used

Reply:
Yes fixed - I added a test on the user for 'scoped' - and a setter that we can use in the tooling to reset it with zero values.